.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -84,11 +94,17 @@ do {
                 newVersion = `${currentPatchVersion}-${highestBuildNumber + 1}`;
                 updateNativeVersions(newVersion);
                 console.log(`Setting npm version for this PR to ${newVersion}`);
-                return exec(`npm version ${newVersion} --force -m "Update version to ${newVersion}"`);
+                return exec(`npm --no-git-tag-version version ${newVersion} -m "Update version to ${newVersion}"`);
             })
             .then(({stdout}) => {
-                // NPM and native versions successfully updated - don't retry.
+                // NPM and native versions successfully updated, create a tag
                 console.log(stdout);
+                const addCommand = `git add ${
+                    [PACKAGE_JSON_PATH, PACKAGE_LOCK_PATH, BUILD_GRADLE_PATH, PLIST_PATH, PLIST_PATH_TEST].join(' ')
+                }`;
+                const commitCommand = `git commit -m "Update version to ${newVersion}`;
+                const tagCommand = `git tag ${newVersion} && git push --tags`;
+                return exec(`${[addCommand, commitCommand, tagCommand].join(' && ')}`);
```

Comment:
Not sure I have an opinion either way, but is there a reason for doing this in JS as opposed to in the action?

Reply:
The only reason would be to use the new version in the commit message and tag. However, one of the things on our project roadmap is having this action output the new version, so we could just output the new version from JS then do the git stuff in the workflow.

Reply:
Discussed offline and I think we both agreed that moving this out of JS will make our lives and code easier in the long run